components/x-live-blog-post/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@financial-times/x-content-block",
+  "name": "@financial-times/x-live-blog-post",
   "version": "0.0.0",
   "description": "",
-  "main": "dist/ContentBlock.cjs.js",
-  "module": "dist/ContentBlock.esm.js",
-  "browser": "dist/ContentBlock.es5.js",
-  "style": "dist/ContentBlock.css",
+  "main": "dist/LiveBlogPpost.cjs.js",
```

Comment:
Small typo - there is an extra `p` here that needs to be removed

Reply:
Thanks Keran 👍 

Should be fixed now.